src/test/java/marquez/db/dao/TestOwnerDAO.java

```diff
@@ -10,10 +15,51 @@
   @ClassRule public static final DAOSetup daoSetup = new DAOSetup();
 
   final OwnerDAO ownerDAO = daoSetup.onDemand(OwnerDAO.class);
+  final Owner testOwner = new Owner("Amaranta");
+
+  @Before
+  public void setUp() {
+    daoSetup
+        .getJDBI()
+        .useHandle(
+            handle -> {
+              handle.execute("DELETE FROM ownerships");
+              handle.execute("DELETE FROM owners");
+            });
+  }
+
+  private static String ownerDeletedAt(String name) {
+    List<String> deletedAts =
+        daoSetup
+            .getJDBI()
+            .withHandle(
+                handle ->
+                    handle
+                        .createQuery("SELECT deleted_at FROM owners WHERE name = :name")
+                        .bind("name", name)
+                        .mapTo(String.class)
+                        .list());
```

Comment:
Use `findOnly()` instead of `list()` to return a single user name given that only one result is possible. Also, maybe we want to bind the result to an `Owner` object instead of a string?